### PR TITLE
Lesson 5 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-5-(Unit-Testing).md [hunk 1]
```diff
@@ -10,11 +10,10 @@
...
 
 1. How to place tests in the correct location within a csharp project
-1. How to use the ``@Test`` annotation to mark a test method
-1. How to use ``@Before`` to generate test data to be used by each test within a class
-1. The behavior of ``@After``
-1. How to run JUnit tests as a group, or individually, within IntelliJ
-1. How to use common assertion methods: ``assertEquals``, ``assertFalse``, ``assertTrue``, ``assertNotNull``
+1. How to use the ``[TestMethod]`` annotation to mark a test method and the ``[TestClass]`` annotation to mark a test class.
+1. How to use ``[TestInitialize]`` to generate test data to be used by each test within a class.
+1. How to run MSTest tests as a group, or individually, within IntelliJ
```
Comment (CarlyLanglois) [inline Class-5-(Unit-Testing).md]: Visual Studio

File: Class-5-(Unit-Testing).md [hunk 2]
```diff
@@ -27,28 +26,29 @@
...
    * Grouping related tests
 1. Talk about tests that use AAAs, are deterministic, relevant, and meaningful
-1. csharp annotations - what are they and what do they do
-   * ``@Test``
-   * ``@Before``
-   * ``@After``
+1. C# annotations - what are they and what do they do
+   * ``[TestClass]``
+   * ``[TestMethod]``
+   * ``[TestInitialize]``
```
Comment (CarlyLanglois) [inline Class-5-(Unit-Testing).md:37]: testcleanup too

File: Class-5-(Unit-Testing).md [hunk 3]
```diff
@@ -27,28 +26,29 @@
...
 1. Running test files and running single tests
 
 ## Small Group Time: Class 5 Studio (TF Notes)
 
 1. This studio asks students to write tests for a broken method
-1. Students must fork the started code and start an IntelliJ project from Git
+1. Students must fork the started code and start a Visual Studio solution from Git
```
Comment (CarlyLanglois) [inline Class-5-(Unit-Testing).md]: starter code*

**Review by CarlyLanglois — APPROVED** [review]

File: Class-5-(Unit-Testing).md (hunk 1 above)
Comment (gildedgardenia) [reply Class-5-(Unit-Testing).md]: Done!

File: Class-5-(Unit-Testing).md (hunk 2 above)
Comment (gildedgardenia) [reply Class-5-(Unit-Testing).md:37]: Added!

File: Class-5-(Unit-Testing).md (hunk 3 above)
Comment (gildedgardenia) [reply Class-5-(Unit-Testing).md]: Done!

**Pull request merged**